app/models/user.rb

```diff
@@ -177,11 +175,16 @@ def update_stats
   end
 
   # Override Devise::Model::Lockable#lock_access! to add event logging
-  def lock_access!
-    EventLog.record_event(User.find_by_email(self.email), EventLog::ACCOUNT_LOCKED)
-    super
-
-    UserMailer.locked_account_explanation(self).deliver_later
+  def lock_access! opts = {}
+    event = case locked_reason
+            when :passphrase
+              EventLog::ACCOUNT_LOCKED
+            when :two_step
+              EventLog::TWO_STEP_LOCKED
+            end
+
+    EventLog.record_event(self, event)
+    super opts
```

Comment:
Calling `super` as a bare word passes the enclosing arguments by default. If you want to golf this down a little.